app/api/domains/cho.py

```diff
@@ -636,17 +684,12 @@ async def login(
             player.logout()
             del player
 
-    user_info = await players_repo.fetch_one(
-        name=login_data["username"],
-        fetch_all_fields=True,
-    )
-
+    user_info = await authenticate(login_data["username"], login_data["password_md5"])
     if user_info is None:
-        # no account by this name exists.
         return {
-            "osu_token": "unknown-username",
+            "osu_token": "incorrect-credentials",
             "response_body": (
-                app.packets.notification(f"{BASE_DOMAIN}: Unknown username")
+                app.packets.notification(f"{BASE_DOMAIN}: Incorrect credentials")
```

Comment:
incorrect username vs. incorrect password will now both return "incorrect credentials"

Reply:
heh, i halfway agree but this is definitely kinda funny since all profiles are publicly searchable by username